test/db/analysis/msp430

```diff
@@ -1,7 +1,7 @@
 NAME=aFr on ELF
 FILE=bins/elf/msp430.elf
 CMDS=<<EOF
-s main
+s loc.main
```

Comment:
Why this?

Reply:
The old behavior was buggy. Indeed, to detect the entry, the function was returning the offset without checking if it was a vaddr or a paddr. So the `s main` was jumping on a vaddr not a paddr (for this binary). I removed the `is_vaddr` so no entry is detected (msp430 is a “broken”).

Reply:
This is strange. I'd still expect to have `main` and not `loc.main`, because there is even a `main` symbol in the binary.

Reply:
Okay, I will fix this. And in the next PR, I will try to remove the "hack".